modules/core/shared/src/main/scala/gem/math/Angle.scala

```diff
@@ -410,7 +419,7 @@ object HourAngle extends HourAngleOptics {
    * @group Constructors
    */
   def fromDoubleHours(hs: Double): HourAngle =
-    fromMicroseconds((hs * 60.0 * 60.0 * 1000.0).toLong)
+    fromMicroseconds((hs * µsPerHour).round)
```

Comment:
The bug was just that we were off by 1000, but I also changed this to `round`.  Is there any reason why we should just truncate instead?

Reply:
No, I didn't know about `.round`. That's fine.

We can't define round-trip optics for doubles because in general they never work. That's another area we could explore I guess.